x-pack/plugins/apm/public/components/app/Main/AccessControl/InvalidLicenseNotification.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiButton, EuiEmptyPrompt } from '@elastic/eui';
+import React from 'react';
+import chrome from 'ui/chrome';
+
+const MANAGE_LICENSE_URL = `${chrome.getBasePath()}/app/kibana#/management/elasticsearch/license_management`;
+
+export const InvalidLicenseNotification = () => (
+  <EuiEmptyPrompt
+    iconType="alert"
+    iconColor="warning"
+    title={<h1>Invalid license</h1>}
+    body={<p>Your current license does not support the Kibana APM UI.</p>}
```

Comment:
I think we can improve this message slightly, as any Kibana license (if active) will support APM UI. How about;

`APM UI is not available because your current license has expired or is no longer valid.`

Thoughts?

Reply:
Oh yeah that makes way more sense. Thanks! :+1:

Reply:
This License warning looks so much better than what we had before. Thanks!